packages/fast-component-explorer/app/explorer.tsx

```diff
@@ -373,22 +491,30 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
 
     private handleUpdateDirection = (): void => {
         this.setState({
-            // The viewConfig only contains the direction,
-            // if more items are added to the viewConfig use merge
-            viewConfig: {
+            viewConfig: merge({}, this.state.viewConfig, {
                 direction:
                     this.state.viewConfig.direction === Direction.ltr
                         ? Direction.rtl
                         : Direction.ltr,
-            },
+                designSystem: Object.assign({}, this.state.viewConfig.designSystem, {
+                    direction:
+                        this.state.viewConfig.direction === Direction.ltr
+                            ? Direction.rtl
+                            : Direction.ltr,
```

Comment:
theoretically we can now simplify this to remove direction and just use the design system, we shouldnt need to determine it twice